docs/features/models/kbqa.rst

```diff
@@ -250,8 +249,8 @@ To use Entity Linking service in KBQA, in the :config:`kbqa_cq_en <kbqa/kbqa_cq_
         "class_name": "api_requester",
         "id": "entity_linker",
         "url": "entity_linking_url",
-        "out": ["entity_ids"],
-        "param_names": ["entity_substr", "template_found"]
+        "out": ["entity_substr", "entity_ids", "entity_conf", "entity_pages", "entity_labels"],
+        "param_names": ["entity_substr", "tags", "probas", "sentences"]
```

Comment:
Doesn't fit entity_linking_en.json input and output. Or should fit other config interface?